app/src/main/java/ch/epfl/sdp/drone3d/mission/MappingMissionService.kt

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (C) 2021  Drone3D-Team
+ * The license can be found in LICENSE at root of the repository
+ */
+
+package ch.epfl.sdp.drone3d.mission
+
+import ch.epfl.sdp.drone3d.drone.DroneService
+
+class MappingMissionService constructor(val droneService: DroneService) {
```

Comment:
Add an @Inject to DroneService such that Hilt will know that it should retreive the droneService from its context when constructing this.

Reply:
Something like this should work (not 100% sure)

```suggestion
class MappingMissionService constructor(val droneService: DroneService) {

@Module
@InstallIn(SingletonComponent::class)
abstract class MappingMissionModule {
    
    @Singleton
    @Binds
    abstract fun bindMappingMissionService(impl: MappingMissionService): MappingMissionService 
}
```

Reply:
Done